shell basic for loop.txt

```diff
@@ -0,0 +1,8 @@
+#Basic for loop to print Hello world 5 times
+
+for i in 1..5
```

Comment:
you can even give a ; after 1..5 and a space before do to reduce the number of lines